docker-registry-sync/Makefile

```diff
@@ -33,19 +35,41 @@ dev-dependencies: _ensure-in-venv ## install development depenencies in your ven
 
 .PHONY: compile-dependencies
 compile-dependencies: _ensure-in-venv ## snapshot dependencies
-	@pip-compile --build-isolation $(current_dir)/reposync/requirements/base.in --output-file $(current_dir)/reposync/requirements/base.txt 
-	@pip-compile --build-isolation $(current_dir)/reposync/requirements/test.in --output-file $(current_dir)/reposync/requirements/test.txt 
+	@pip-compile --build-isolation reposync/requirements/base.in --output-file reposync/requirements/base.txt 
+	@pip-compile --build-isolation reposync/requirements/test.in --output-file reposync/requirements/test.txt 
 
 
 .PHONY: install-dependencies
 install-dependencies: _ensure-in-venv ## install package dependencies
-	@pip install -r $(current_dir)/reposync/requirements/base.txt 
+	@cd reposync/requirements; pip-sync base.txt
+
+.PHONY: install-test-dependencies
+install-test-dependencies: _ensure-in-venv ## install package test dependencies
+	@cd reposync/requirements; pip-sync test.txt 
 
 
 .PHONY: install-py-package
 install-py-package: _ensure-in-venv ## install package dependencies
 	@pip install -e reposync
 
+.PHONY: tests
+tests: _ensure-in-venv ## install package dependencies
+	@tox -r -c reposync/tox.ini
+
+
+.PHONY: setup-env
+setup-env: ## creates the .env still need to change your credentials
```

Comment:
minor: you could use a recipe named directly .env and then use $@ to write to it.
or actually use a default .env.default